content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -9,13 +9,16 @@ weight: 65
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
 
 Dynamic resource allocation is an API for requesting and sharing resources
 between pods and containers inside a pod. It is a generalization of the
 persistent volumes API for generic resources. Third-party resource drivers are
-responsible for tracking and allocating resources. Different kinds of
-resources support arbitrary parameters for defining requirements and
+responsible for tracking and allocating resources, with additional support
+provided by Kubernetes via "structured parameters" (introduced in Kubernetes 1.30).
```

Comment:
```suggestion
provided by Kubernetes via _structured parameters_ (introduced in Kubernetes 1.30).
```

Please also consider adding “structured parameter” into our glossary.

Reply:
I applied your suggestion and squashed (forgot that earlier).

I'll keep the glossary in mind for the next update. Right now I feel it's too early - we still need to find out whether the name and concept survives.